bin/user/ambientweatherapi.py

```diff
@@ -218,6 +220,31 @@ def genLoopPackets(self):
 					error_occured = False
 					raise Exception('Previous error occured, skipping packet build.')
 
+				# Read previous daily rain total, and write most recent daily rain back to file
+				if path.exists('rain.txt') == True:
+					print('Opening file')
+					intervalRain = open('/Users/Shared/weewx/rain.txt', 'r')
```

Comment:
This pathing won't work on most systems.  i.e. my Fedora box does not have such a path.

Reply:
Ah sorry, forgot to update that from when I was debugging. Just pushed a commit to fix

Reply:
Actually, when I run that updated code on my system, I get an error about the file system being read-only. It looks like the script is trying to write rain.txt to '/', instead of the weewx root. I'm not sure how to pull the WEEWX_ROOT variable into the script so I can use os.path.join() to combine it with 'rain.txt'.

Reply:
Never mind—figured it out a different way. The script just gets its own location and uses that as the working directory.